mitmproxy/tools/console/flowlist.py

```diff
@@ -35,19 +35,21 @@ def keypress(self, size, key):
 
 
 class FlowListWalker(urwid.ListWalker):
+    __cache = {}
 
     def __init__(self, master):
         self.master = master
 
     def positions(self, reverse=False):
         # The stub implementation of positions can go once this issue is resolved:
         # https://github.com/urwid/urwid/issues/294
-        ret = range(self.master.commands.execute("view.properties.length"))
```

Comment:
As @madt1m noted, not accessing the view addon here was intentional. The overhead this currently causes is not really justifiable though. Looking a bit further down the profile graph, it seems like we could significantly alleviate this by lru-caching `command.verify_arg_signature` (14.61%) and `command.lexer` (5.3%)? That would also benefit other parts of mitmproxy. :)